cli/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
I understand your thoughts @matiasbeckerle. I don't see a bad thing to have it, as a separate project the generator side.

I'm not sure that there's a `good way` or `the way`, (It's a really old discussion but I think it's not an important thing to spend time) but if I need to say something I may prefer the 4 spaces. More clear and neat (for me obviously). 🖖